utils/calendrical_calculations/src/helpers.rs

```diff
@@ -351,270 +199,7 @@ fn test_invert_angular() {
-            expected: 16.5,
-        },
-        TestCase {
-            x: 9.0,
-            a: 3.0,
-            b: 9.0,
-            expected: 3.0,
-        },
-        TestCase {
-            x: 17.0,
-            a: 1.0,
-            b: 5.5,
-            expected: 3.5,
-        },
-    ];
-
-    for case in cases {
-        let result = interval_mod_f64(case.x, case.a, case.b);
-        assert_eq!(
-            case.expected, result,
-            "Interval mod test failed for case: {case:?}"
-        );
-    }
-}
-
-#[test]
-fn test_div_rem_euclid() {
-    assert_eq!(div_rem_euclid(i32::MIN, 1), (-2147483648, 0));
-    assert_eq!(div_rem_euclid(i32::MIN, 2), (-1073741824, 0));
-    assert_eq!(div_rem_euclid(i32::MIN, 3), (-715827883, 1));
-
-    assert_eq!(div_rem_euclid(-10, 1), (-10, 0));
-    assert_eq!(div_rem_euclid(-10, 2), (-5, 0));
-    assert_eq!(div_rem_euclid(-10, 3), (-4, 2));
-
-    assert_eq!(div_rem_euclid(-9, 1), (-9, 0));
-    assert_eq!(div_rem_euclid(-9, 2), (-5, 1));
-    assert_eq!(div_rem_euclid(-9, 3), (-3, 0));
-
-    assert_eq!(div_rem_euclid(-8, 1), (-8, 0));
-    assert_eq!(div_rem_euclid(-8, 2), (-4, 0));
-    assert_eq!(div_rem_euclid(-8, 3), (-3, 1));
-
-    assert_eq!(div_rem_euclid(-2, 1), (-2, 0));
-    assert_eq!(div_rem_euclid(-2, 2), (-1, 0));
-    assert_eq!(div_rem_euclid(-2, 3), (-1, 1));
-
-    assert_eq!(div_rem_euclid(-1, 1), (-1, 0));
-    assert_eq!(div_rem_euclid(-1, 2), (-1, 1));
-    assert_eq!(div_rem_euclid(-1, 3), (-1, 2));
-
-    assert_eq!(div_rem_euclid(0, 1), (0, 0));
-    assert_eq!(div_rem_euclid(0, 2), (0, 0));
-    assert_eq!(div_rem_euclid(0, 3), (0, 0));
-
-    assert_eq!(div_rem_euclid(1, 1), (1, 0));
-    assert_eq!(div_rem_euclid(1, 2), (0, 1));
-    assert_eq!(div_rem_euclid(1, 3), (0, 1));
-
-    assert_eq!(div_rem_euclid(2, 1), (2, 0));
-    assert_eq!(div_rem_euclid(2, 2), (1, 0));
-    assert_eq!(div_rem_euclid(2, 3), (0, 2));
-
-    assert_eq!(div_rem_euclid(8, 1), (8, 0));
-    assert_eq!(div_rem_euclid(8, 2), (4, 0));
-    assert_eq!(div_rem_euclid(8, 3), (2, 2));
-
-    assert_eq!(div_rem_euclid(9, 1), (9, 0));
-    assert_eq!(div_rem_euclid(9, 2), (4, 1));
-    assert_eq!(div_rem_euclid(9, 3), (3, 0));
-
-    assert_eq!(div_rem_euclid(10, 1), (10, 0));
-    assert_eq!(div_rem_euclid(10, 2), (5, 0));
-    assert_eq!(div_rem_euclid(10, 3), (3, 1));
-
-    assert_eq!(div_rem_euclid(i32::MAX, 1), (2147483647, 0));
-    assert_eq!(div_rem_euclid(i32::MAX, 2), (1073741823, 1));
-    assert_eq!(div_rem_euclid(i32::MAX, 3), (715827882, 1));
-
-    for n in -100..100 {
-        for d in 1..5 {
-            let (x1, y1) = div_rem_euclid(n, d);
-            let (x2, y2) = div_rem_euclid64(n as i64, d as i64);
-            let (x3, y3) = (n.div_euclid(d), n.rem_euclid(d));
-            assert_eq!(x1, x2 as i32);
-            assert_eq!(x1, x3);
-            assert_eq!(y1, y2 as i32);
-            assert_eq!(y1, y3);
-        }
-    }
-}
-
-#[test]
-fn test_ceil_div() {
```

Comment:
Please restore tests for functions that are implemented in this file

Reply:
I'll mark PRs as ready when they are ready to review